lib/AttachmentProcessor.js

```diff
@@ -5,7 +5,11 @@ const REQUEST_TIMEOUT = process.env.REQUEST_TIMEOUT ? parseInt(process.env.REQUE
 const REQUEST_MAX_RETRY = process.env.REQUEST_MAX_RETRY ? parseInt(process.env.REQUEST_MAX_RETRY, 10) : 7; // 10s
 const REQUEST_RETRY_DELAY = process.env.REQUEST_RETRY_DELAY ? parseInt(process.env.REQUEST_RETRY_DELAY, 10) : 7000; // 7s
 const REQUEST_MAX_CONTENT_LENGTH = process.env.REQUEST_MAX_CONTENT_LENGTH ? parseInt(process.env.REQUEST_MAX_CONTENT_LENGTH, 10) : 10485760; // 10MB
-const { ATTACHMENT_STORAGE_SERVICE_BASE_URL } = process.env;
+const ATTACHMENT_STORAGE_SERVICE_BASE_URL = process.env.ATTACHMENT_STORAGE_SERVICE_BASE_URL || 'http://attachment-storage-service.oih-dev-ns.svc.cluster.local:3002';
```

Comment:
This assumes it's in an oih-dev-ns namespace. The problem with setting it this way is that it works great in a lower environment and then fails in production. I'd rather the fallback be oih-prod-ns.